.github/copilot-instructions.md

```diff
@@ -0,0 +1,268 @@
+# Decidim - Participatory Democracy Framework
+
+Decidim is a Ruby on Rails application with JavaScript frontend components, supporting multiple modules for participatory democracy processes. The codebase consists of multiple gem modules and requires specific setup procedures.
+
+**ALWAYS reference these instructions first and fallback to search or bash commands only when you encounter unexpected information that does not match the info here.**
+
+## Working Effectively
+
+### System Requirements
+- **CRITICAL**: Ruby **3.3.4** (NOT 3.2.3 - will fail with dependency errors)
+- **CRITICAL**: Node.js **22.14.0** (NOT 20.x - specified in .node-version)
```

Comment:
@andreslucena , would be better to instruct the pilot to read the .ruby-version and .node-version files ?

Reply:
Agree